fvm/transactionInvoker.go

```diff
@@ -275,6 +276,21 @@ func (executor *transactionExecutor) normalExecution() (
 	modifiedSets programsCache.ModifiedSetsInvalidator,
 	err error,
 ) {
+	executor.txnState.RunWithAllLimitsDisabled(func() {
```

Comment:
move RunWithAllLimitsDisabled into TransactionPayerBalanceChecker / CheckPayerBalanceAndReturnMaxFees.  The executor doesn't need to know about this detail.